webapp/maps_heatmaps/src/clickInfoWindow.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { map } from "./index";
+import { getQueriedCollection } from "./queryDB";
+import * as firebase from "firebase";
+import { GeoFirestoreTypes } from "geofirestore";
+
+//TODO: run mocha tests on the browser.
+let markers: Array<google.maps.Marker> = [];
+
+/* querys for 20 random docs in the database in order to place markers on them. */
+//TODO: make this function more random.
+//TODO: use this function to show images on the side pannel-so they will correlate (relocate to a different file)
+async function setFirstTwentyMarkers(
+  center: firebase.firestore.GeoPoint,
+  radius: number,
+  labels: string[],
+  year?: number,
+  month?: number
+): Promise<void> {
+  let lastInfoWindow: google.maps.InfoWindow | null = new google.maps.InfoWindow();
+  const dataref = await (
+    await getQueriedCollection(center, radius, labels, year, month).get()
+  ).docs;
+  eraseAllMarkers();
+  const jump = Math.ceil(dataref.length / 10);
+  for (let i = 0; i < dataref.length; i = i + jump) {
+    addMarkerWithListener(
+      map,
+      convertGeopointTolatlon(dataref[i].data().g.geopoint),
```

Comment:
Use lowerCamelCase, i.e. convertGeopointToLatLon, see:
go/tsstyle-oss-interns?cl=320047486#identifiers

Reply:
Changed convertGeopointTolatlon to convertGeopointToLatLon.